src/main/java/seedu/duke/command/sprint/AllocateSprintTaskCommand.java

```diff
@@ -32,8 +32,8 @@ public void execute() {
         allSprint = proj.getAllSprints();
         if (allSprint.updateCurrentSprint()) {
             if (validateParams()) {
-                int taskId = Integer.parseInt(this.parametersInHT.get("task").trim());
-                String[] userIds = this.parametersInHT.get("user").split(" ");
+                int taskId = Integer.parseInt(this.parameters.get("task").trim());
```

Comment:
Should we trim the values at `Parser` instead?

Reply:
Sure. Did I simple update on parser to facilitate this.